app/services/activities/import_from_csv.rb

```diff
@@ -203,14 +211,15 @@ def set_errors
     class ActivityCreator
```

Comment:
Does this need applying to `ActivityUpdater` too?

Reply:
I think it doesn't, since where `ActivityCreator` calls `return true if @activity.save(context: @validation_steps)`, `ActivityUpdater` just calls `return set_errors unless @activity.save` - no context argument to pass in the validation steps